microprofile/metrics/src/main/java/io/helidon/microprofile/metrics/MetricsCdiExtension.java

```diff
@@ -445,7 +450,8 @@ private void processInjectionPoints(@Observes ProcessInjectionPoint<?, ?> pip) {
      * @param pat the {@code ProcessAnnotatedType} for the type containing the JAX-RS annotated methods
      */
     private void recordSimplyTimedForRestResources(@Observes
-                                                   @WithAnnotations({HttpMethod.class})
+                                                   @WithAnnotations({GET.class, PUT.class, POST.class, HEAD.class, OPTIONS.class,
```

Comment:
(This is fine; when I commented on this previously, I didn't mean necessarily to remove the `HttpMethod` annotation and replace it with the list of options—it was more just that I was surprised that a meta-annotation like that would be processed by `@WithAnnotations`.)

Reply:
Yeah, the original code had the list of individual annotations and that worked fine. I happened to see `@HttpMethod` in the def for `Get` (I think) and thought I'd try it and it seemed to work. But that is not related to the functional change at the heart of this PR so it's probably best to not make that change...at least now.